heft-plugins/heft-sass-plugin/src/schemas/heft-sass-plugin.schema.json

```diff
@@ -74,7 +88,7 @@
 
     "importIncludePaths": {
```

Comment:
Possibly I should just delete this from the schema like I did for `preserveSCSSExtension`?